src/plugins/httpUpload/HttpUploadPlugin.ts

```diff
@@ -158,7 +158,7 @@ export default class HttpUploadPlugin extends AbstractPlugin {
 
       let html = $('<div>').append(message.getHtmlMessage());
 
-      let linkElement = $('<a>');
+      let linkElement = $('<a target="_blank">');
```

Comment:
We need to add `rel="noopener noreferrer"` for privacy and security protection.

Reply:
added